src/core/bitop.d

```diff
@@ -72,20 +72,18 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
+{
 /// ditto
+int bsf(ulong v) pure;
+}
+else version(D_LP64) { } else
```

Comment:
> This is because size_t can be either 32 bit or 64 bit and there is no plans to support anything else

Plans change.

> (and if there were such plans something like D_LP32 version would be added at first).

To the standard `version` specifiers, but not to this code. Everyone will have forgotten about this conversation by then.